### PR TITLE
feature: enhance-comments-list-api-by-django-filter

### PR DESCRIPTION
**Pull request by W-Shih**
- feature: enhance-comments-list-api-by-django-filter
  - filter by tweet_id and user_id
- test: add more tests for comments-list-api

### CODE REVIEW

File: comments/api/tests.py [hunk 1]
```diff
@@ -224,14 +225,26 @@ def test_list_api(self):
...
+        response = self.anonymous_client.get(COMMENT_URL, {
+            'tweet_id': 1000000
+        })
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+        # Non-existing user id
+        response = self.anonymous_client.get(COMMENT_URL, {
+            'user_id': 1000000
+        })
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
```
Comment (wen-wayne-shih) [inline comments/api/tests.py:238]: use non_existing_user_id / non_existing_tweet_id instead

**Review by wen-wayne-shih — CHANGES_REQUESTED** [review]
must be addressed before merging.

File: comments/api/tests.py (hunk 1 above)
Comment (W-Shih) [reply comments/api/tests.py]: resolved as suggestions

**Review by wen-wayne-shih — APPROVED** [review]
squash before merging

**Pull request merged**